src/rust/include/rust/orchard/wallet.h

```diff
@@ -366,7 +368,7 @@ OrchardSpendInfoPtr* orchard_wallet_get_spend_info(
         const OrchardWalletPtr* wallet,
         const unsigned char txid[32],
         uint32_t action_idx,
-        const unsigned char as_of_root[32]);
+        size_t confirmations);
```

Comment:
Update the comment to say something like "relative to the root of the Orchard commitment tree with the given number of confirmations."